docs/source/index.rst

```diff
@@ -3,14 +3,13 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
+|RAT banner|
+
 .. _RAT: 
 
 Welcome to RAT's documentation!
```

Comment:
Use the RAT banner here, I think you can remove the "Welcome to RAT's documentation!"

Reply:
Just to clarify - You want to include the RAT banner only in index.rst ?

Reply:
Yes for now, we could place in other places going forward